unstructured/ingest/connector/local.py

```diff
@@ -48,6 +50,19 @@ def cleanup_file(self):
     def get_file(self):
         """Not applicable to local file system"""
 
+    def update_source_metadata(self, **kwargs) -> None:
+        try:
+            out = os.lstat(self.path)
+            self._source_metadata = SourceMetadata(
+                exists=True,
+                date_created=str(datetime.fromtimestamp(out.st_ctime)),
+                date_modified=str(datetime.fromtimestamp(out.st_mtime)),
+                permissions_data=[{"mode": out.st_mode}],
```

Comment:
reasoning / value on this?

Reply:
this is the file mode data which I would consider the permissions associated with the file.